microsite/src/main/tut/Laws.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: docs
+title:  "Laws"
+---
+
+# Laws [![GitHub](../img/github.png)]
```

Comment:
Oh, that reminds me that I need to port the 7.3 changes here :-/

Reply:
Ooh, which?

Reply:
#1949 I "automated" some stuff, pages have automatic "Edit" links and link to their source on GitHub